actionpack/test/controller/base_test.rb

```diff
@@ -27,6 +27,16 @@ def default_url_options
   end
 end
 
+class DefaultFormatController < ActionController::Base
+  def show
+    render :nothing => true
```

Comment:
@tekin We decided to only change the hash notation within the patch diff itself. The reasoning behind it is that just changing the hash notation makes the git blame history harder to follow as well as changes harder to backport. that's why we still have a lot of hash rockets around but try to replace them within the PR-diffs. Hope that makes sense ;)